C/Cpp_include/c4Collection.hh

```diff
@@ -104,6 +104,7 @@ struct C4Collection : public fleece::RefCounted, C4Base, fleece::InstanceCounted
 
     using CollectionObserverCallback = std::function<void(C4CollectionObserver*)>;
     using DocumentObserverCallback = std::function<void(C4DocumentObserver*,
+                                                        C4CollectionSpec,
```

Comment:
It seems better to pass a C4Collection pointer than a C4CollectionSpec. It's smaller, and the platform code can probably map it to its own thing more efficiently than having to look up two strings.

Reply:
@pasin Let's see what the platform owners think about which is easier